src/components/DayPickerKeyboardShortcuts.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { PropTypes } from 'react';
+import { forbidExtraProps } from 'airbnb-prop-types';
+
+import { DayPickerKeyboardShortcutsPhrases } from '../defaultPhrases';
+import getPhrasePropTypes from '../utils/getPhrasePropTypes';
+
+import CloseButton from '../svg/close.svg';
+
+const propTypes = forbidExtraProps({
+  showKeyboardShortcutsPanel: PropTypes.bool,
+  toggleKeyboardShortcutsPanel: PropTypes.func,
+  phrases: PropTypes.shape(getPhrasePropTypes(DayPickerKeyboardShortcutsPhrases)),
+});
+
+const defaultProps = {
+  showKeyboardShortcutsPanel: false,
+  toggleKeyboardShortcutsPanel() {},
+  phrases: DayPickerKeyboardShortcutsPhrases,
+};
+
+function KeyboardShortcutRow({ unicode, label, action }) {
+  return (
+    <li className="KeyboardShortcutRow">
+      <div
+        className="KeyboardShortcutRow__key-container"
+      >
+        <span className="KeyboardShortcutRow__key" aria-label={label}>
```

Comment:
Should this be `<kbd>` instead of span? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd